api/open_audio_pool.go

```diff
@@ -0,0 +1,55 @@
+package api
```

Comment:
this feels a lot more bloaty than we need.

i would just have set an array of OpenAudioSdk in the config.go file and then just pick randomly when we need it